node_4.x.dockerfile

```diff
@@ -8,10 +8,10 @@ RUN apt-get-install.sh apt-transport-https curl && \
     echo "deb https://deb.nodesource.com/node_4.x jessie main" \
         > /etc/apt/sources.list.d/nodesource.list && \
     apt-get-purge.sh curl
-ENV NODEJS_VERSION "4.4.2"
-RUN apt-get-install.sh nodejs=${NODEJS_VERSION}*
 
-ENV VXSANDBOX_VERSION "0.6.0"
+RUN apt-get update && apt-get install curl -y && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install nodejs -y
```

Comment:
What is the reasoning behind installing node this way? It's a lot safer installing using apt-get than piping a script from the internet into bash.

Reply:
The apt-get command was failing, so I got this from casepro.